examples/movingtarget_example_data/moving_target_test.yaml

```diff
@@ -55,7 +55,7 @@ simSignals:
   psfpixfrac: 0.25                           #Fraction of a pixel between entries in PSF library (e.g. 0.1 = files for PSF centered at 0.1 pixel intervals within pixel)
   psfwfe: predicted                               #PSF WFE value (0,115,123,132,136,150,155)
   psfwfegroup: 0                             #WFE realization group (0 to 9)
-  galaxyListFile: movingtarget_example_data/moving_galaxies.list
```

Comment:
I think I may have gotten myself confused yesterday and mentioned the wrong catalogs for this example. This notebook creates a scene assuming non-sidereal tracking of a target. So `movingTargetToTrack` should be populated with `movingtarget_example_data/moving_target_to_track.cat`. Then the other catalogs will provide background (sidereal) sources that are going to be trailed in the exposure. So `pointsource` should be set to `imaging_example_data/seed_im_from_catalog_test_ptsrc_catalog.list`. The `movingTargetList`, `movingTargetSersic`, and `movingTargetExtended` should all be set to None. If you wanted trailed ellipses in the output, then you could set `galaxyListFile` to `imaging_example_data/seed_im_from_catalog_test_galaxies_catalog.list`.

The `movingtarget_example_data/moving_galaxies.list` catalog comes into play for a sidereal exposure where you want to have ellipses trailing through the background. In this case you can specify a different velocity for each target, whereas in the non-sidereal case above, the velocity is the same for all background sources.

I'm running the notebook now with the catalogs I mentioned above, but it takes a long time because there are so many sources. Maybe I should update the example catalogs so that they have only a couple sources in them.

Reply:
Done!